support/hololens/ServoApp/ServoControl/Servo.cpp

```diff
@@ -87,6 +87,10 @@ const char *prompt_input(const char *message, const char *default,
   }
 }
 
+void on_devtools_started(const unsigned int port) {
+  return sServo->Delegate().OnDevtoolsStarted(port);
```

Comment:
Do you have access to a Windows machine to test the UWP compilation or you'd like me to do that for you?

Reply:
No, I don't have a Windows machine. You'll have to do that for me.

Reply:
Sure thing! I'll take care of that.

Reply:
For now - remove the call to `Delegate().OnDevtoolsStarted`. Just add a `// FIXME`. I will add a proper UI to notify the user in another PR.